### ISSUE TITLE
Inquiry about the test set of PP-HumanSeg14K dataset

### ISSUE THREAD

**Issue by kisungsim** [PaddlePaddle/PaddleSeg#3086]
### 问题确认 Search before asking

- [X] 我已经搜索过问题，但是没有找到解答。I have searched the question and found no related answer.

### 请提出你的问题 Please ask your question

I have a question about the test set of PP-HumanSeg14K dataset.
I would be grateful if you answer my question.
A few months ago, I sent an e-mail through paddleseg@baidu.com whether I can get the PP-HumanSeg14K dataset.
Thankfully, your team sent me the PP-HumanSeg14K dataset link.
But, I found that there exist annotation files in the train set and validation set, but there is no annotation file in the test set.
So, I want to ask whether I can get the annotation files of the test set of PP-HumanSeg14K.
I'm looking forward to your reply.
Best Regards,

**Comment by juncaipeng** [comment]
The annotation files of test set are not provided externally. You can use the validation set to test the performance.

**Comment by kisungsim** [comment]
Thank you for your reply :)
Have a good day!

**Comment by langxuelang** [comment]
I also sent an email to the official team requesting access to the PP-HumanSeg14K dataset. Unfortunately, I haven't received any response after waiting for quite some time.

Would it be possible for you to share the dataset with me? I would greatly appreciate your help.

My email address is: [gaotengxuelang@gmail.com](mailto:gaotengxuelang@gmail.com)
